packages/ui/src/PasswordFieldFormik.tsx

```diff
@@ -7,6 +7,7 @@ import { ExtractKeysOfValueType } from './utils/types'
 
 export type PasswordFieldFormikProps<V extends object> = PasswordFieldExtraProps & {
   name: ExtractKeysOfValueType<V, string | undefined>
+  id?: string
```

Comment:
Is it really passed by formik or is it just an extra prop?

Reply:
Not entirely sure what do you mean. Could you elaborate please?